src/participant_network/cl/lighthouse/lighthouse_launcher.star

```diff
@@ -99,8 +99,14 @@ def launch(
 	beacon_service = add_service(beacon_node_service_id, beacon_config)
 
 	# TODO check whether its 200, 206 or 503 like golang
-	define_fact(service_id = beacon_node_service_id, fact_name = BEACON_HEALTH_FACT_NAME, fact_recipe = struct(method= "GET", endpoint = "/eth/v1/node/health", content_type = "application/json", port_id = BEACON_HTTP_PORT_ID))
-	wait(service_id = beacon_node_service_id, fact_name = BEACON_HEALTH_FACT_NAME)
+	health_recipe = struct(
```

Comment:
could we use src/participant_network/cl/cl_node_health_checker.star here?

Reply:
good catch